src/infrastructure/database/utils/prepared-statements.ts

```
@@ -86,18 +99,28 @@
           const result = await client.query(query, params);
           const duration = Date.now() - startTime;
-          QueryLoggerService.logQuery(`PREPARED ${name}: ${query}`, params, duration, 'postgresql');
+          QueryLoggerService.logQuery(
+            `PREPARED ${name}: ${query}`,
+            params,
+            duration,
+            'postgresql'
+          );
 
           // Update usage statistics
           if (this.cache[name]) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)